libs/sdk-ui-dashboard/src/presentation/dashboard/Dashboard.tsx

```diff
@@ -66,35 +64,39 @@ import { IDashboardProps } from "./types";
 
 const useFilterBar = (): {
     filters: FilterContextItem[];
-    onFilterChanged: (filter: FilterContextItem | undefined) => void;
+    onAttributeFilterChanged: (filter: IDashboardAttributeFilter) => void;
+    onDateFilterChanged: (filter: IDashboardDateFilter | undefined, dateFilterOptionLocalId?: string) => void;
```

Comment:
Can filter be undefined?

Reply:
Yes, if All time date filter is selected.